fv3core/stencils/fxadv.py

```diff
@@ -1,314 +1,381 @@
+        damp_v = 1.0 / (1.0 - 0.0625 * cosa_u * cosa_v)
+        vt = (
+            vc
+            - 0.25
+            * cosa_v
+            * (
+                ut[1, 0, 0]
+                + ut[1, -1, 0]
+                + ut[0, -1, 0]
+                + uc
+                - 0.25 * cosa_u * (vt[0, 1, 0] + vt[-1, 1, 0] + vt[-1, 0, 0])
+            )
+        ) * damp_v
+    return vt
 
 
-def compute_vt(uc_in, vc_in, cosa_v, rsin_v, sin_sg2, sin_sg4, vt_in):
-    vt_origin = (grid().isd, grid().js - 1, 0)
-    vt = utils.make_storage_from_shape(vt_in.shape, vt_origin)
-    main_vt(
-        uc_in,
-        vc_in,
-        cosa_v,
-        rsin_v,
-        vt,
-        origin=vt_origin,
-        domain=(grid().nid, grid().njc + 3, grid().npz),
-    )  # , origin=(0, 2, 0), domain=(vt.shape[0]-1, main_j_size, vt.shape[2]))
-
-    # Cannot pass vt_in array to stencil without it zeroing out data outside
-    # specified domain. So... for now copying in so the 'undefined' answers
-    # match
-    vt[:, : grid().js - 1, :] = vt_in[:, : grid().js - 1, :]
-    vt[:, grid().je + 3, :] = vt_in[:, grid().je + 3, :]
-    if grid().south_edge:
-        vt[:, grid().js, :] = vt_in[:, grid().js, :]
-    if grid().north_edge:
-        vt[:, grid().je + 1, :] = vt_in[:, grid().je + 1, :]
-    return vt
+@gtscript.function
+def ra_x_func(area, xfx_adv):
+    return area + xfx_adv - xfx_adv[1, 0, 0]
 
 
-def update_vt_y_edge(vc, cosa_v, ut, vt):
-    if grid().west_edge or grid().east_edge:
-        j1 = grid().js + 2 if grid().south_edge else grid().js
-        j2 = grid().je if grid().north_edge else grid().je + 2
-        edge_shape = (2, j2 - j1, ut.shape[2])
-        if grid().west_edge:
-            vt_y_edge(
-                vc, cosa_v, ut, vt, origin=(grid().is_ - 1, j1, 0), domain=edge_shape
-            )
-        if grid().east_edge:
-            vt_y_edge(vc, cosa_v, ut, vt, origin=(grid().ie, j1, 0), domain=edge_shape)
-
-
-def update_vt_x_edge(vc, sin_sg2, sin_sg4, vt, dt):
-    if grid().south_edge or grid().north_edge:
-        edge_shape = (vt.shape[0], 1, vt.shape[2])
-        if grid().south_edge:
-            vt_x_edge(
-                vc,
-                sin_sg2,
-                sin_sg4,
-                vt,
-                dt=dt,
-                origin=(0, grid().js, 0),
-                domain=edge_shape,
-            )
-        if grid().north_edge:
-            vt_x_edge(
-                vc,
-                sin_sg2,
-                sin_sg4,
-                vt,
-                dt=dt,
-                origin=(0, grid().je + 1, 0),
-                domain=edge_shape,
-            )
+@gtscript.function
+def ra_y_func(area, yfx_adv):
+    return area + yfx_adv - yfx_adv[0, 1, 0]
 
 
-# -------------------- CORNERS-----------------
+# -------------------- DEPRECATED CORNERS-----------------
+"""
+# Using 1 function with different sets of externals
+# Unlikely to use as different externals in single stencil version
```

Comment:
Not sure how I feel about this, but this could be better as a comment instead of block string.

Reply:
I mentioned in the PR definition that it was extremely helpful to have the original version commented here to switch already, it saved me a lot of time and effort, so am motivated to keep around this 'single stencil' version for a bit. What is the benefit to making every line start with #? I could see grepping for the code, and thinking it is live code, could be one pitfall of not making it a comment.